target/linux/ipq40xx/image/mikrotik.mk

```diff
@@ -19,10 +19,21 @@ define Device/mikrotik_hap-ac2
 endef
 TARGET_DEVICES += mikrotik_hap-ac2
 
+define Device/mikrotik_ldf-5-ac
+	$(call Device/mikrotik_nor)
+	DEVICE_MODEL := LDF 5 ac
+	SOC := qcom-ipq4018
+	DEVICE_PACKAGES := ipq-wifi-mikrotik_ldf-5-ac -kmod-ath10k-ct \
+		kmod-ath10k-ct-smallbuffers rssileds -kmod-usb2 -kmod-usb3 \
+		-libusb-1.0
+endef
+TARGET_DEVICES += mikrotik_ldf-5-ac
+
 define Device/mikrotik_sxtsq-5-ac
 	$(call Device/mikrotik_nor)
 	DEVICE_MODEL := SXTsq 5 ac (RBSXTsqG-5acD)
 	SOC := qcom-ipq4018
 	DEVICE_PACKAGES := ipq-wifi-mikrotik_sxtsq-5-ac rssileds
 endef
 TARGET_DEVICES += mikrotik_sxtsq-5-ac
+
```

Comment:
...

Reply:
ok. I seem to have a tendency to end files with an empty line ... :-) Thanks for review!

Reply:
For now, keep this request on hold please.
While testing over short distances works fine, that is not the case over larger distances (test site uses 7 km).
Two issues: 
1. receive sensitivity: it turns out this is cured by using the regular ath10k driver, not the ct version. Difference about 10 dB!
So using the ct version, like in the hAP ac2 device, has issues.
2. distance/coverage class setting is not working properly. No fix yet. iw appears not capable of setting this.
Will request review once resolved.